internal/mediainfo/dvd.go

```
@@ -426,25 +579,33 @@ func parseDVDVideo(path string, file *os.File, size int64, opts AnalyzeOptions)
 
-	if menuStream != nil {
+	for i := range menuStreams {
 		if aggregateMode {
 			if source := dvdTitleSetSource(base); source != "" {
-				appendCanonicalSeedObjectMembers(menuStream, "extra", []structuredMember{{
+				appendCanonicalSeedObjectMembers(&menuStreams[i], "extra", []structuredMember{{
 					Key:   "Source",
 					Value: structuredNode{Kind: structuredString, Text: source},
 				}})
 
 			}
```

golangci-lint (2.12.2)

[error] unnecessary trailing newline

(whitespace)